apps/condo/domains/user/schema/ForgotPasswordService.js

```diff
@@ -121,17 +120,18 @@ const ForgotPasswordService = new GQLCustomSchema('ForgotPasswordService', {
                 if (password.length < MIN_PASSWORD_LENGTH) {
                     throw new Error(`${PASSWORD_TOO_SHORT}] Password is too short`)
                 }
-                const forgotPasswordData = await ForgotPasswordActionUtil.getAll(context, {
+                const [action] = await ForgotPasswordActionUtil.getAll(context, {
                     token,
                     expiresAt_gte: now,
                 })
 
-                if (isEmpty(forgotPasswordData) || !forgotPasswordData.length) {
+                if (!action || action.usedAt) {
                     throw new Error(`${RESET_TOKEN_NOT_FOUND}] Unable to find token`)
```

Comment:
Maybe this way, to be consistent:
```suggestion
                    throw new Error(`${RESET_TOKEN_NOT_FOUND}] Unable to find valid token`)
```

Reply:
Done